From 7d9f0181909990574a7bcb99a079e43676fab260 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 26 Oct 2012 01:37:54 -0700
Subject: [PATCH 2/3] Add patch to change boot delay to 0 seconds

---
 build.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/build.sh b/build.sh
index b2c7736..882248a 100755
--- a/build.sh
+++ b/build.sh
@@ -282,6 +282,9 @@ build_u_boot () {
 		git am "${DIR}/patches/v2012.10/0001-Exynos-Add-minimal-support-for-ODROID-X.patch"
 	fi
 
+	# Patches for Mecanum Bot
+	git am "${DIR}/../patches/Bootloader-Builder/u-boot/0001-Change-boot-delay-to-0-seconds.patch"
+
 	if [ -f "${DIR}/stop.after.patch" ] ; then
 		echo "-----------------------------"
 		pwd
-- 
1.7.11

