From f24993e54c2e8f4954f295e6f7cf7f1785c159fc Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 26 Oct 2012 01:38:46 -0700
Subject: [PATCH 3/3] Add patch to set BeagleBoard xM pin mux

---
 build.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build.sh b/build.sh
index 882248a..5692437 100755
--- a/build.sh
+++ b/build.sh
@@ -284,6 +284,7 @@ build_u_boot () {
 
 	# Patches for Mecanum Bot
 	git am "${DIR}/../patches/Bootloader-Builder/u-boot/0001-Change-boot-delay-to-0-seconds.patch"
+	git am "${DIR}/../patches/Bootloader-Builder/u-boot/0002-Configure-BeagleBoard-xM-pin-mux.patch"
 
 	if [ -f "${DIR}/stop.after.patch" ] ; then
 		echo "-----------------------------"
-- 
1.7.11

