From 607c5579be6070da9721c4cf50789af62c13a445 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 26 Oct 2012 01:58:36 -0700
Subject: [PATCH 2/7] Include additional packages specified in settings.xml

---
 var/pkg_list.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/var/pkg_list.sh b/var/pkg_list.sh
index d4450a9..a6ecd39 100755
--- a/var/pkg_list.sh
+++ b/var/pkg_list.sh
@@ -8,6 +8,9 @@ MINIMAL_APT="${MINIMAL_APT},openssh-server,apache2,ntpdate,ppp"
 MINIMAL_APT="${MINIMAL_APT},btrfs-tools,cpufrequtils,fbset"
 MINIMAL_APT="${MINIMAL_APT},initramfs-tools,lsb-release"
 
+# Mecanum dependencies (__ MECANUM_PACKAGES __ is replaced by list of packages)
+MINIMAL_APT="${MINIMAL_APT},__MECANUM_PACKAGES__"
+
 #not in squeeze/oneiric
 precise_wheezy_plus=",wvdial"
 
-- 
1.7.11

