From 09b436f5b4d4abd37c05479528155d1149a6e62f Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 26 Oct 2012 02:01:16 -0700
Subject: [PATCH 4/7] Remove text from /etc/flash-kernel.conf

---
 tools/fixup.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/fixup.sh b/tools/fixup.sh
index f359140..9a43701 100755
--- a/tools/fixup.sh
+++ b/tools/fixup.sh
@@ -24,7 +24,6 @@ cat > /etc/flash-kernel.conf <<-__EOF__
 	#!/bin/sh -e
 	UBOOT_PART=/dev/mmcblk0p1
 
-	echo "flash-kernel stopped by: /etc/flash-kernel.conf"
 	USE_CUSTOM_KERNEL=1
 
 	if [ "\${USE_CUSTOM_KERNEL}" ] ; then
-- 
1.7.11

