From 3d97ef220b6deb2114f2795b5151df6682db249e Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 26 Oct 2012 02:02:58 -0700
Subject: [PATCH 5/7] Copy keys to the new filesystem

---
 tools/fixup.sh | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/tools/fixup.sh b/tools/fixup.sh
index 9a43701..282d7d1 100755
--- a/tools/fixup.sh
+++ b/tools/fixup.sh
@@ -85,3 +85,17 @@ rm -f /tmp/*.deb || true
 rm -rf /usr/src/linux-headers* || true
 rm -f /rootstock-user-script || true
 
+
+# Copy ssh keys to new filesystem
+# Note: permissions still need to be set to 600 (with USER_NAME as the owner)
+# Parameters below will be filled in by create_script.py
+#USER_NAME=__USER_NAME__
+if [ "\$USER_NAME" ] ; then
+mkdir -p /home/${USER_NAME}/.ssh
+cat > /home/${USER_NAME}/.ssh/id_rsa <<RSA
+__RSA_PRIVATE__
+RSA
+cat > /home/${USER_NAME}/.ssh/id_rsa.pub <<RSA
+__RSA_PUBLIC__
+RSA
+fi
-- 
1.7.11

