From 240eeba918ba22eaacbc9c8c4c3d7d5579433c62 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Sun, 3 Jun 2012 04:50:58 -0700
Subject: [PATCH 5/7] Copy keys to the new filesystem

---
 tools/fixup.sh |   15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/tools/fixup.sh b/tools/fixup.sh
index 98e5be0..55dfdf0 100755
--- a/tools/fixup.sh
+++ b/tools/fixup.sh
@@ -96,3 +96,18 @@ rm -f /tmp/*.deb || true
 rm -rf /usr/src/linux-headers* || true
 rm -f /rootstock-user-script || true
 
+
+# Copy ssh keys to new filesystem
+# Note: permissions still need to be set to 600 (with USER_NAME as the owner)
+# Parameters below will be filled in by create_script.py
+#USER_NAME=__USER_NAME__
+if [ "\$USER_NAME" ] ; then
+mkdir -p /home/${USER_NAME}/.ssh
+cat > /home/${USER_NAME}/.ssh/id_rsa <<RSA
+__RSA_PRIVATE__
+RSA
+cat > /home/${USER_NAME}/.ssh/id_rsa.pub <<RSA
+__RSA_PUBLIC__
+RSA
+fi
+
-- 
1.7.9.5

