From bf2b1a5d7d29c1414b876c769103b245e2d86107 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 26 Oct 2012 02:27:58 -0700
Subject: [PATCH 6/7] Primary kernel is on local filesystem

---
 build_image.sh | 12 +++++++-----
 1 file changed, 7 insertions(+), 5 deletions(-)

diff --git a/build_image.sh b/build_image.sh
index c1bb154..6c09682 100755
--- a/build_image.sh
+++ b/build_image.sh
@@ -79,6 +79,8 @@ function dl_rootstock {
 
 	git checkout origin/third-kernel -b run-script
 
+	git am "${DIR}/../patches/omap-image-builder/project-rootstock/0001-Primary-kernel-is-on-local-filesystem.patch"
+
 	cd ${DIR}/deploy/
 }
 
@@ -203,7 +205,7 @@ function select_rcn-ee-net_kernel {
 	SUBARCH="omap"
 	KERNEL_ABI="TESTING"
 	kernel_chooser
-	PRIMARY_KERNEL="--kernel-image ${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DEB_FILE}"
+	PRIMARY_KERNEL="--kernel-image __KERNEL_DEB_FILE__"
 	echo "Using: ${PRIMARY_KERNEL}"
 	unset PRIMARY_DTB_FILE
 	if [ "x${ACTUAL_DTB_FILE}" != "x" ] ; then
@@ -220,11 +222,11 @@ function select_rcn-ee-net_kernel {
 	SUBARCH="omap-psp"
 	KERNEL_ABI="STABLE"
 	kernel_chooser
-	SECONDARY_KERNEL="--secondary-kernel-image ${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DEB_FILE}"
+	#SECONDARY_KERNEL="--secondary-kernel-image ${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DEB_FILE}"
 	echo "Using: ${SECONDARY_KERNEL}"
 	unset SECONDARY_DTB_FILE
 	if [ "x${ACTUAL_DTB_FILE}" != "x" ] ; then
-		SECONDARY_DTB_FILE="${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DTB_FILE}"
+		#SECONDARY_DTB_FILE="${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DTB_FILE}"
 		echo "Using dtbs: ${SECONDARY_DTB_FILE}"
 	fi
 
@@ -237,11 +239,11 @@ function select_rcn-ee-net_kernel {
 	SUBARCH="omap"
 	KERNEL_ABI="STABLE"
 	kernel_chooser
-	THIRD_KERNEL="--third-kernel-image ${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DEB_FILE}"
+	#THIRD_KERNEL="--third-kernel-image ${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DEB_FILE}"
 	echo "Using: ${THIRD_KERNEL}"
 	unset THIRD_DTB_FILE
 	if [ "x${ACTUAL_DTB_FILE}" != "x" ] ; then
-		THIRD_DTB_FILE="${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DTB_FILE}"
+		#THIRD_DTB_FILE="${DEB_MIRROR}/${DIST}-${ARCH}/${FTP_DIR}/${ACTUAL_DTB_FILE}"
 		echo "Using dtbs: ${THIRD_DTB_FILE}"
 	fi
 }
-- 
1.7.11

