From dfb902ef5dc8555c9a094143cf0d3856f6aec06b Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 26 Oct 2012 02:30:11 -0700
Subject: [PATCH 7/7] Run script to install ros

---
 tools/fixup.sh | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/tools/fixup.sh b/tools/fixup.sh
index 282d7d1..b7594ef 100755
--- a/tools/fixup.sh
+++ b/tools/fixup.sh
@@ -99,3 +99,12 @@ cat > /home/${USER_NAME}/.ssh/id_rsa.pub <<RSA
 __RSA_PUBLIC__
 RSA
 fi
+
+
+# Run our script to install ROS and set up the environment for our Mecanum Bot
+cd /tmp/
+git clone git://github.com/garbear/mecanum.git
+cd mecanum/arm/scripts/
+python installRosMobile.py
+cd /tmp/
+rm -rf /tmp/mecanum/
-- 
1.7.11

