From 72fb1117614434404457660d678c2e039915eeb9 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 18 May 2012 10:27:19 -0700
Subject: [PATCH 1/3] Only build Precise image

---
 build_image.sh |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/build_image.sh b/build_image.sh
index 09e2910..56ea5b9 100755
--- a/build_image.sh
+++ b/build_image.sh
@@ -427,14 +427,14 @@ SECONDARY_KERNEL_SEL="STABLE"
 #SECONDARY_KERNEL_SEL="TESTING"
 #SECONDARY_KERNEL_SEL="EXPERIMENTAL"
 
-ARCH=armel
-oneiric_release
-if [ "-${HOST_ARCH}-" == "-armv7l-" ] ; then
-squeeze_release
-fi
+#ARCH=armel
+#oneiric_release
+#if [ "-${HOST_ARCH}-" == "-armv7l-" ] ; then
+#squeeze_release
+#fi
 
 ARCH=armhf
 precise_release
-wheezy_release
+#wheezy_release
 
 
-- 
1.7.9.5

