From 57e0014145f7bac8f7cbd759ea8e33424445e8d1 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 18 May 2012 11:04:41 -0700
Subject: [PATCH 2/7] Include additional packages specified in settings.xml

---
 build_image.sh |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/build_image.sh b/build_image.sh
index d3a7941..858174b 100755
--- a/build_image.sh
+++ b/build_image.sh
@@ -107,6 +107,9 @@ MINIMAL_APT="${MINIMAL_APT},openssh-server,apache2"
 MINIMAL_APT="${MINIMAL_APT},btrfs-tools,usb-modeswitch,wireless-tools,wpasupplicant"
 MINIMAL_APT="${MINIMAL_APT},cpufrequtils,fbset,ntpdate,ppp"
 
+# Mecanum dependencies (__ MECANUM_PACKAGES __ is replaced by list of packages)
+MINIMAL_APT="${MINIMAL_APT},__MECANUM_PACKAGES__"
+
 #Hostname:
 FQDN="omap"
 
-- 
1.7.9.5

