From 0ba3e405d7799770916130f337d8572f37ef5a7a Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Fri, 18 May 2012 11:08:14 -0700
Subject: [PATCH 3/3] Force MAC address

---
 tools/fixup.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/fixup.sh b/tools/fixup.sh
index 7951854..a62d4a8 100755
--- a/tools/fixup.sh
+++ b/tools/fixup.sh
@@ -12,7 +12,7 @@ echo "# The primary network interface" >> /etc/network/interfaces
 echo "auto eth0"  >> /etc/network/interfaces
 echo "iface eth0 inet dhcp"  >> /etc/network/interfaces
 echo "# Example to keep MAC address between reboots"  >> /etc/network/interfaces
-echo "#hwaddress ether DE:AD:BE:EF:CA:FE"  >> /etc/network/interfaces
+echo "hwaddress ether DE:AD:BE:EF:CA:FE"  >> /etc/network/interfaces
 echo "" >> /etc/network/interfaces
 echo "# WiFi Example" >> /etc/network/interfaces
 echo "#auto wlan0" >> /etc/network/interfaces
-- 
1.7.9.5

