From c8b9cba8df13a22f01561e4ff4c92eaa676b6da4 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Sat, 26 May 2012 15:50:59 -0700
Subject: [PATCH 5/7] Run script to install ros

---
 tools/fixup.sh |    7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/tools/fixup.sh b/tools/fixup.sh
index 98e5be0..0bb53c2 100755
--- a/tools/fixup.sh
+++ b/tools/fixup.sh
@@ -96,3 +96,10 @@ rm -f /tmp/*.deb || true
 rm -rf /usr/src/linux-headers* || true
 rm -f /rootstock-user-script || true
 
+# Run our script to install ROS and set up the environment for our Mecanum Bot
+cd /tmp/
+git clone git://github.com/garbear/mecanum.git
+cd mecanum/arm/
+/usr/bin/env python arm_install_ros.py
+cd -
+rm -rf /tmp/mecanum/
-- 
1.7.9.5

