From 27c11502ac58c87d10889bf558858e717f335fa1 Mon Sep 17 00:00:00 2001
From: Garrett Brown <garbearucla@gmail.com>
Date: Thu, 31 May 2012 01:26:25 -0700
Subject: [PATCH 6/7] Copy ssh keys to the new filesystem

---
 tools/fixup.sh |   13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/tools/fixup.sh b/tools/fixup.sh
index 0bb53c2..c2f2f0c 100755
--- a/tools/fixup.sh
+++ b/tools/fixup.sh
@@ -103,3 +103,16 @@ cd mecanum/arm/
 /usr/bin/env python arm_install_ros.py
 cd -
 rm -rf /tmp/mecanum/
+
+# Copy ssh keys to new filesystem
+# Parameters below will be filled in by create_script.py
+#USER_NAME=__USER_NAME__
+if [ "\$USER_NAME" ] ; then
+mkdir -p /home/${USER_NAME}/.ssh
+cat > /home/${USER_NAME}/.ssh/id_rsa <<RSA
+__RSA_PRIVATE__
+RSA
+cat > /home/${USER_NAME}/.ssh/id_rsa.pub <<RSA
+__RSA_PUBLIC__
+RSA
+# Note: permissions still need to be set to 600 (with USER_NAME as the owner)
-- 
1.7.9.5

